maps/tether/submaps/offmap/talon1.dmm

```diff
@@ -216,9 +219,76 @@
 /obj/machinery/chemical_dispenser/full,
 /turf/simulated/floor/tiled/eris/white/bluecorner,
 /area/talon/deckone/medical)
+"ax" = (
+/obj/machinery/camera/network/talon,
+/obj/structure/closet/wardrobe/black{
+	starts_with = list(/obj/item/clothing/head/helmet/combat = 4, /obj/item/clothing/mask/gas/commando = 4, /obj/item/clothing/mask/balaclava = 4, /obj/item/clothing/under/syndicate/combat = 4, /obj/item/clothing/accessory/storage/black_vest = 4, /obj/item/clothing/accessory/storage/black_drop_pouches = 4, /obj/item/clothing/gloves/combat = 4, /obj/item/clothing/suit/armor/combat = 4, /obj/item/clothing/shoes/boots/combat = 4)
```

Comment:
I don't think this counts as a "minor edit", this is giving the talon four full sets of combat armor, while currently they get a single helmet, armor suit, and jumpsuit. Those gloves are heavily armored and insulated, while the boots are noslips. Why does the talon need to be armored up like they're nuke ops?

Reply:
> 
> 
> Generally rule of thumb is that unless you are intentionally re balancing things, you should generally stick with the same amount of gear between versions

Gotcha!